Rakefile

```diff
@@ -1,4 +1,24 @@
 require 'rspec/core/rake_task'
 
-RSpec::Core::RakeTask.new(:spec)
-task :default => :spec
+namespace :py do
+  load "./py/Rakefile"
+end
+
+namespace :rb do
+  desc "Install Ruby dependencies"
+  task :install do
+    require 'rubygems'
+    require 'bundler/setup'
+  end
+
+  RSpec::Core::RakeTask.new(:test)
```

Comment:
This line is failing because it is trying to use RSpec::Core::RakeTask before rspec is installed.

Might be a way to rewrite this - i.e. something like this:

```
task "rb:test" do
  RSpec::Core::RakeTask.new(:test).run
end
```

here the RS::C:RakeTask is only loaded when the test task is run. I reckon this should work